tests/validate_plusless_properties.py

```diff
@@ -1,6 +1,12 @@
 import json 
 from jsondiff import diff
 
+PROJECT_SCHEMA_FILES = [
+    "./schemas/dbt_project.json",
+    "./schemas/dbt_project-1.5.json",
+    "./schemas/dbt_project-1.6.json"
+]
+
```

Comment:
This might be better as a query that lists all files matching the pattern `dbt_project[\-\d\.]*\.json` or something, instead of a hardcoded list that we have to remember to keep up to date